src/pages/Home/components/TodayPick/TodayPick.style.ts

```diff
@@ -0,0 +1,166 @@
+import { css } from '@emotion/react';
+
+import { theme } from '@/styles/theme';
+
+// ===== [1] Wrapper & Title Row =====
+export const wrapper = css`
+  width: 100%;
+  max-width: 37.5rem;
+  margin: 0 auto;
+  padding: 0 1.2rem;
+`;
+
+export const titleRow = css`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0.8rem 0.6rem;
+`;
+
+export const best = css`
+  ${theme.fonts.title1};
+  color: ${theme.colors.black1};
+`;
+
+export const moreButton = css`
+  display: inline-flex;
+  align-items: center;
+  gap: 0.4rem;
+  width: 5.7rem;
+  height: 2rem;
+  background: none;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+`;
+
+export const moreText = css`
+  ${theme.fonts.subtitle4};
+  color: ${theme.colors.black1};
+  white-space: nowrap;
+`;
+
+// ===== [2] Tabs =====
+
+export const tabBlock = css`
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+`;
+
+export const tabItem = css`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+  height: auto;              
+  padding: 0 1.2rem 0.8rem; 
```

Comment:
```suggestion
  padding-bottom: 0.8rem; 
```
으로 해주고 TodayPick.tsx에서 태그를 묶어주는 방법을 조금만 바꾸면 완벽할 것 같아요 ㅎㅎ